crates/pallet-humanode-session/src/lib.rs

```diff
@@ -72,6 +75,66 @@ pub mod pallet {
     #[pallet::storage]
     pub type CurrentSessionIndex<T: Config> = StorageValue<_, SessionIndex, OptionQuery>;
 
+    /// A list of all banned accounts that can't be validators in the network.
+    #[pallet::storage]
+    pub type BannedAccounts<T: Config> =
+        StorageValue<_, BoundedVec<T::AccountId, T::MaxBannedAccounts>, ValueQuery>;
```

Comment:
Our use of the storage item indicates it would be much better as a set instead of a vec

Reply:
https://github.com/humanode-network/humanode/pull/1249/commits/5f9e7bbb234e3a3192de8a7a45475cb77fa26043